core/src/main/scala/org/apache/spark/sql/rapids/tool/util/MemoryMetricsTracker.scala

```diff
@@ -0,0 +1,32 @@
+package org.apache.spark.sql.rapids.tool.util
```

Comment:
nit: is there a reason why there is no license header here?

Reply:
@cindyyuanjiang Thanks for pointing out ! Updated. Have also updated the README with working changes for running classes directly